lib/push.js

```diff
@@ -253,190 +253,148 @@ module.exports = function (log, db, config) {
 
   return {
 
-    isValidPublicKey: isValidPublicKey,
+    isValidPublicKey,
 
     /**
-     * Notifies all devices that there was an update to the account
+     * Notify devices that there was an update to the account
```

Comment:
`notifyUpdate` now does all of the work that `pushToAllDevices` and `pushToDevices` used to do. By passing in the `devices` array everywhere, it left us with a layer of indirection that wasn't doing anything. I opted to keep `notifyUpdate` and get rid of the other two because its name matches the other method names exported from this module.